### PR TITLE
Migration of all tests to golang `testing` & `testify` asserts

### PR DESCRIPTION
**Pull request by stefannegrea**
Coarse migration of all tests from `gospec` to pure golang `testing` & `testify` asserts.

### CODE REVIEW

File: enqueue_test.go [hunk 1]
```diff
@@ -2,118 +2,104 @@ package workers
...
-		})
-	})
+	retryCount := int(result["retry_count"].(float64))
+	assert.Equal(t, 13, retryCount)
+}
 
-	c.Specify("EnqueueIn", func() {
-		scheduleQueue := "prod:" + SCHEDULED_JOBS_KEY
-		rc := Config.Client
+func TestEnqueueSpec(t *testing.T) {
```
Comment (rbutler) [inline enqueue_test.go]: should this say `TestEnqueueIn`?

File: manager_test.go [hunk 2]
```diff
@@ -35,165 +35,176 @@ func (m *customMid) Trace() []string {
...
+	assert.Equal(t, int64(0), len)
+	assert.True(t, drained)
+}
 
-			len, _ := rc.LLen("prod:queue:manager1").Result()
-			c.Expect(len, Equals, int64(0))
-			c.Expect(drained, Equals, true)
-		})
+func TestMultiMiddleware(t *testing.T) {
+	//per-manager middlwares are called separately, global middleware is called in each manager", func() {
```
Comment (rbutler) [inline manager_test.go]: leftover comment?

File: enqueue_test.go (hunk 1 above)
Comment (stefannegrea) [reply enqueue_test.go]: Good catch; thank you!

File: manager_test.go (hunk 2 above)
Comment (stefannegrea) [reply manager_test.go]: I would like to keep the comment but the `func` at the end needs to go.

**Pull request merged**